v63003/inref/gtm4212.m

```diff
@@ -10,61 +10,32 @@
 ;								;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;
+; Creates 4 paths based on the current directory, of length 230, 231
+; <=230 and >=231 and stores them in temp files
 ;
-;
-path231
-	;assumes path is already less than 231 characters long
-	set bigstring="temp231.out"
-	open bigstring
-	use bigstring write $ZDIRECTORY
-	set l=$length($ZDIRECTORY)
-	use bigstring write "a"
-	set l=l+1
-	if l<231 do  goto path231+8
-	. if l#25=0 use bigstring write "/"
-	. else  use bigstring write "a"
-	. set l=l+1
-	quit
-
-path230
-	;assumes path is already less than 230 characters long
-	set bigstring="temp230.out"
-	open bigstring
-	use bigstring write $ZDIRECTORY
-	set l=$length($ZDIRECTORY)
-	use bigstring write "a"
-	set l=l+1
-	if l<230 do  goto path230+8
-	. if l#25=0 use bigstring write "/"
-	. else  use bigstring write "a"
-	. set l=l+1
-	quit
-
-pathge231
-	set x=231+$random(10)+1
-	set bigstring="tempge231.out"
-	open bigstring
-	use bigstring write $ZDIRECTORY
-	set l=$length($ZDIRECTORY)
-	use bigstring write "a"
-	set l=l+1
-	if l<x do  goto pathge231+8
-	. if l#25=0 use bigstring write "/"
-	. else  use bigstring write "a"
-	. set l=l+1
+gtm4212
+	set p230="temp230.out"
```

Comment:
Lines 17-28 can be simplified by storing the various choices in one variable (e.g. set filelist="temp230.out temp231.out ..." and using $piece(filelist) in a for loop.